src/apps/atlendis-v1/polygon/atlendis-v1.balance-fetcher.ts

```diff
@@ -0,0 +1,53 @@
+import { Inject } from '@nestjs/common';
+import { BigNumber } from 'ethers';
+
+import { drillBalance } from '~app-toolkit';
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { presentBalanceFetcherResponse } from '~app-toolkit/helpers/presentation/balance-fetcher-response.present';
+import { BalanceFetcher } from '~balance/balance-fetcher.interface';
+import { Network } from '~types/network.interface';
+
+import { ATLENDIS_V_1_DEFINITION } from '../atlendis-v1.definition';
+
+const network = Network.POLYGON_MAINNET;
+
+@Register.BalanceFetcher(ATLENDIS_V_1_DEFINITION.id, network)
+export class PolygonAtlendisV1BalanceFetcher implements BalanceFetcher {
+  constructor(@Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit) {}
+
+  async getPositionBalances(address: string) {
+    return this.appToolkit.helpers.contractPositionBalanceHelper.getContractPositionBalances({
```

Comment:
Replace the body of `getPositionBalances` with the following logic:

1. Make a request to https://atlendis.herokuapp.com/graphql, requesting the positions as follows:

```gql
query {
  positions(address:"0x634ed8C2dF8fBe613eE25DEBE1a460B4DF2487d7") {
    tokenId
    normalizedAmount
    pool{
      parameters{
        underlyingToken
      }
    }
  }
}
```

(You'll find examples of the helper classes for making GQL queries here: https://studio.zapper.fi/docs/recipes/graphql-helper)

2. Once you have the token IDs, you can iterate over them, and build contract positions as:

```ts
const positions = data.positions.map(position => {
  const underlyingToken = baseTokens.find(
    t => t.address.toLowerCase() === position.pool.parameters.underlyingToken.toLowerCase(),
  );
  if (!underlyingToken) return null;

  const position = {
    type: ContractType.POSITION,
    appId,
    groupId,
    address: positionManagerAddress,
    network,
    tokens: [supplied(underlyingToken)],
    dataProps: {
      tokenId: position.tokenId,
      amount: position.normalizedAmount,
      owner: position.lender.address,
    },
    displayProps: {
      label: getLabelFromToken(underlyingToken),
      images: getImagesFromToken(underlyingToken),
    },
  };

  const tokenBalances = [drillBalance(position.tokens[0]), BigNumber.from(position.normalizedAmount).div(wad).mul(tokenDecimals).toString())]
  const contractPositionBalance: ContractPositionBalance = {
    ...position,
    tokens: tokenBalances,
    balanceUSD: sumBy(tokenBalances, v => v.balanceUSD),
  }

  return contractPositionBalance;
});
```